# Papers on Machine Learning in Chemistry

Papers on machine learning in quantum chemistry, cheminformatics, drug discovery, and materials science

---

# Table of Contents
- [Papers on Machine Learning in Chemistry](#papers-on-machine-learning-in-chemistry)
- [Table of Contents](#table-of-contents)
- [Gentle Introduction to Machine Learning (Deep Learning)](#gentle-introduction-to-machine-learning-deep-learning)
- [Quantum Chemistry](#quantum-chemistry)
  - [Energy and Force (DFT)](#energy-and-force-dft)
  - [Models](#models)
  - [Packages](#packages)
- [Drug Discovery](#drug-discovery)
  - [Deep Learning Review Articles](#deep-learning-review-articles)
  - [Generative Adversarial Networks](#generative-adversarial-networks)
  - [Molecular Properties](#molecular-properties)
  - [Representation](#representation)
  - [Multitask Networks](#multitask-networks)
  - [Benchmarks](#benchmarks)
- [Cheminformatics](#cheminformatics)
  - [Structure-Based Virtual Screening](#structure-based-virtual-screening)
  - [Organic Reaction Prediction](#organic-reaction-prediction)
  - [Molecular Graph](#molecular-graph)
  - [Data Mining](#data-mining)
  - [Synthesis Planing](#synthesis-planing)
  - [Molecular Design](#molecular-design)
  - [Image & 3D Structure](#image--3d-structure)
  - [Transfer Learning](#transfer-learning)
- [Materials science](#materials-science)
- [Credits](#credits)

# Gentle Introduction to Machine Learning (Deep Learning)

1. [Deep Learning in Chemistry](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00266) (2019)
2. [Deep Learning for Computational Chemistry](https://onlinelibrary.wiley.com/doi/epdf/10.1002/jcc.24764) (2017)
3. The convergence of artificial intelligence and chemistry for improved drug discovery. [Future Science](https://www.future-science.com/doi/full/10.4155/fmc-2018-0161)
4. A Survey of Multi‐task Learning Methods in Chemoinformatics. [Molecular Informatics](https://onlinelibrary.wiley.com/doi/full/10.1002/minf.201800108)
5. Deep learning: new computational modelling techniques for genomics. [Nature Reviews Genetics](https://www.nature.com/articles/s41576-019-0122-6)
6. Deep Learning in Biomedical Data Science. [Annual Review of Biomedical Data Science](https://www.annualreviews.org/doi/abs/10.1146/annurev-biodatasci-080917-013343)
7. Deep learning for computational biology. [Mol Syst Biol ](https://www.embopress.org/doi/full/10.15252/msb.20156651)

---

# Quantum Chemistry

## Energy and Force (DFT)

1. Atom-centered symmetry functions for constructing high-dimensional neural network potentials, J. Chem. Phys. 134(7), 074106 (2011) [[pdf]](https://doi.org/10.1063/1.3553717)
2. First Principles Neural Network Potentials for Reactive Simulations of Large Molecular and Condensed Systems, Angewandte Chemie International Edition, 56(42), 12828-12840.(2017）[[pdf]](https://doi.org/10.1002/anie.201703114)
3. Machine learning of accurate energy-conserving molecular force fields, Science Advances, 3(5). (2017) [[pdf]](https://doi.org/10.1126/sciadv.1603015)
4. Hierarchical modeling of molecular energies using a deep neural network, J. Chem. Phys.148, 241715 (2018) [[pdf]](https://doi.org/10.1063/1.5011181)
5. ANI-1: An Extensible Neural Network Potential with DFT accuracy at force field computational cost
6. Fast machine learning models of electronic and energetic properties consistently reach approximation errors better than DFT accuracy.
7. Neural Message Passing for Quantum Chemistry.

## Models

1. Comparing the accuracy of high-dimensional neural network potentials and the systematic molecular fragmentation method: A benchmark study fo all-trans alkanes, The Journal of Chemical Physics, 144, 194110 (2016) [[pdf]](https://doi.org/10.1063/1.4950815)
2. High-Dimensional Atomistic Neural Network Potentials for Molecule–Surface Interactions: HCl Scattering from Au(111), J. Phys. Chem. Lett. 8(3), (2017) [[pdf]](https://doi.org/10.1021/acs.jpclett.6b02994)
3. Accurate Neural Network Description of Surface Phonons in Reactive Gas-Surface Dynamics: N2 + Ru(0001), J. Phys. Chem. Lett. 2017, 8, 2131−2136 [[pdf]](https://doi.org/10.1021/acs.jpclett.7b00784)

## Packages

1. An implementation of artifical neural-network potentials for atomistic materials simulations: Performance for TiO2, Computational Materials Science 114 (2016) 135–150 [[pdf]](https://doi.org/10.1016/j.commatsci.2015.11.047)
2. Amp: A modular approach to machine learning in atomistic simulations, Phys. Commun. 207, 310–324 (2016) [[pdf]](https://doi.org/10.1016/j.cpc.2016.05.010)
3. The TensorMol-0.1 model chemistry: a neural network augmented with long-range physics,Chem. Sci., 2018, 9, 2261 [[pdf]](https://doi.org/10.1039/C7SC04934J)

---

# Drug Discovery

## Deep Learning Review Articles

1. Deep Learning for Computational Chemistry. Goh, G. B.; Hoda, N. O.; Vishnu, A. *J. Comput. Chem.* **2017**, DOI: 10.1002/jcc.24764
2. A Renaissance of Neural Networks in Drug Discovery. Baskin, I. I.; Winkler, D.; Tetko, I. V. *Exp. Op. Drug Disc.* **2016**, *11* 785. DOI: 10.1080/17460441
3. Deep Learning in Drug Discovery. Gawehn, E.; Hiss, J. A.; Schneider, G. *Mol. Inf.* **2016**, *35*, 3.
4. Have Artificial Neural Networks Met Expectations in Drug Discovery as Implemented in QSAR framework? Dobchev, D. & Karelson, M. *Expert Opin. Drug Discov.* **2016**, *11*, 627.
5. The Next Era: Deep Learning in Pharmaceutical Research. Ekins, S. *Pharm. Res.* **2016**, *33*, 2594.
6. Machine-learning approaches in drug discovery: methods and applications. Lavecchia, A. *Drug Discov. Today*, **2015**, *20*, 318.
7. Machine learning methods in chemoinformatics. Mitchell, J. B. O. WIREs Comput. *Mol. Sci.* **2014**, *4*, 468.

## Generative Adversarial Networks

1. "The Cornucopia of Meaningful Leads: Applying Deep Adversarial Autoencoders for New Molecule Development in Oncology"

## Molecular Properties

1. "Deep Architectures and Deep Learning in Chemoinformatics: the Prediction of Aqueous Solubility for Drug-like Molecules"
Lusci, A., Pollastri, G. & Baldi, P. J. Chem. Inf. Model. 53, 1563–1575 (2013).

## Representation

1. Convolutional Networks on Graphs for Learning Molecular Fingerprints. David Duvenaud, D.; Maclaurin, D.; Aguilera-Iparraguirre, J.; Gomez-Bombarelli, R.; Hirzel, T.; Aspuru-Guzik, A.; Adams, R. P.
2. Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules. Gómez-Bombarelli, R.; David Duvenaud, D.; Hernandez-Lobato, J. M.; Aguilera-Iparraguirre, J.; Adams, R. P.; Aspuru-Guzik, A.
3. Molecular graph convolutions: moving beyond fingerprints. Kearnes, S. McCloskey, K.; Berndl, M.; Pande, V.; Riley, P. J. Computer Aided Drug Design. **2016**, *30*, 595.
4. Atomic Convolutional Networks for Predicting Protein-Ligand Binding Affinity. Gomes, J.; Ramsundar, B.; Feinberg, E. N.; Pande, V. S. 

## Multitask Networks

1. Multi-task Neural Networks for QSAR Predictions Dahl, G. E.; Jaitly, N.; Salakhutdinov, R. 
2. Deep Neural Nets as a Method for Quantitative Structure-Activity Relationships. Ma, J.; Sheridan, R. P.; Liaw, A.; Dahl, G. E.; Svetnik, V. J. Chem. Inf. Model. 2015, 55, 263.
3. Toxicity Prediction using Deep Learning.
4. Deep Learning as an Opportunity in Virtual Screening. Unterthiner, T.; Mayr, A.; Klambauer, G. NIPS 2014.
5. Massively Multitask Networks for Drug Discovery. 2015. https://arxiv.org/abs/1502.02072
6. "Deep Learning Applications for Predicting Pharamcological Properties of Drugs and Drug Repurposing Using Transcriptomic Data."
7. "Molecular Fingerprint-Baesd Artificial Neural Networks QSAR for Ligand Biological Activity Predictions."

## Benchmarks

1. "MoleculeNet: A Benchmark for Molecular Machine Learning."

---

# Cheminformatics

## Structure-Based Virtual Screening

1. AtomNet: A Deep Convolutional Neural Network for Bioactivity Prediction in Structure-based Drug Discovery. Wallach, I.; Dzamba, M.; Heifets, A. **2015**. arXiv:1510.02855v1
2. NNScore: A Neural-Network-Based Scoring Function for the Characterization of Protein−Ligand Complexes. Durrant, J. D.; McCammon, J. A. *J. Chem. Inf. Model.* **2010**, *50*, 1865.
3. NNScore 2.0: A Neural-Network Receptor–Ligand Scoring Function.
4. Predicting Ligand Binding Modes from Neural Networks Trained on Protein–Ligand Interaction Fingerprints.
5. Protein–Ligand Scoring with Convolutional Neural Networks. Matthew Ragoza, M.; Hochuli, J.; Idrobo, E.; Sunseri, J.; Koes, D. R. *J. Chem. Inf. Model.* **2017**, *57*, 942.
6. Learning Deep Architectures for Interaction Prediction in Structure-based Virtual Screening. Gonczarek, A.; Tomczak, J. M; Zareba, S.; Kaczmar, J.; Dabrowski, P,; Walczak, M. J. arXiv:1610.07187. **2016**

## Organic Reaction Prediction

1. Neural Networks for the Prediction of Organic Chemistry Reactions. Wei, J. N.; Duvenaud, Aspuru-Guzik, A. ACS Cent Sci **2016**, 2, 725.
2. Prediction of Organic Reaction Outcomes Using Machine Learning. Coley, C. W.; Barzilay, R.; Jaakola, T. S.; Green, W. H.; Jensen, K. F. *ACS Cent Sci* **2017**.

## Molecular Graph

1. Sparse hierarchical representation learning on molecular graphs. [arXiv](https://arxiv.org/abs/1908.02065)
2. Deep Generative Models for 3D Compound Design. [BioRxiv](https://www.biorxiv.org/content/10.1101/830497v1)

## Data Mining
  1. Named Entity Recognition and Normalization Applied to Large-Scale Information Extraction from the Materials Science Literature. [J. Chem. Inf. Model. 2019.](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00470) | [ChemRxiv](https://chemrxiv.org/articles/Named_Entity_Recognition_and_Normalization_Applied_to_Large-Scale_Information_Extraction_from_the_Materials_Science_Literature/8226068/1) | [Github](https://github.com/materialsintelligence/matscholar)
  2. Molecular Structure Extraction from Documents Using Deep Learning. [J. Chem. Inf. Model.](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.8b00669)
  3. BioSentVec: creating sentence embeddings for biomedical texts. [IEEE](https://ieeexplore.ieee.org/abstract/document/8904728)
  
## Synthesis Planing
  1. Molecular Transformer: A Model for Uncertainty-Calibrated Chemical Reaction Prediction. [ACS Cent. Sci.2019](https://pubs.acs.org/doi/10.1021/acscentsci.9b00576) | [ChemRxiv](https://chemrxiv.org/articles/Molecular_Transformer_for_Chemical_Reaction_Prediction_and_Uncertainty_Estimation/7297379) | [Github](https://github.com/pschwllr/MolecularTransformer)
  2. Molecular Transformer unifies reaction prediction and retrosynthesis across pharma chemical space. [Chem. Commun. 2019](https://pubs.rsc.org/en/content/articlelanding/2019/CC/c9cc05122h#!divAbstract) 
  
## Molecular Design

1. Re-balancing Variational Autoencoder Loss for Molecule Sequence Generation. (2019) [arXiv](https://arxiv.org/abs/1910.00698)
2. Generating customized compound libraries for drug discovery with machine intelligence (2019) [ChemRxiv](https://chemrxiv.org/articles/Generating_Customized_Compound_Libraries_for_Drug_Discovery_with_Machine_Intelligence/10119299) [Github](https://github.com/ETHmodlab/virtual_libraries)
3. Deep Generative Models for 3D Compound Design. (2019) [BioRxiv](https://www.biorxiv.org/content/10.1101/830497v1) [Github](https://github.com/oxpig/DeLinker)
4. Molecular Generative Model Based On Adversarially Regularized Autoencoder. [arXiv](https://arxiv.org/abs/1912.05617). [Github](https://github.com/gicsaw/ARAE_SMILES)
  
## Image & 3D Structure

1. Learning Drug Function from Chemical Structure with Convolutional Neural Networks and Random Forests. [BioRxiv](https://www.biorxiv.org/content/10.1101/482877v2.full)
2. Optimization of a Deep-Learning Method Based on the Classification of Images Generated by Parameterized Deep Snap a Novel Molecular-Image-Input Technique for Quantitative Structure–Activity Relationship (QSAR) Analysis. [Front. Bioeng. Biotechnol.](https://www.frontiersin.org/articles/10.3389/fbioe.2019.00065/full)

## Transfer Learning

1. Inductive Transfer Learning for Molecular Activity Prediction: Next-Gen QSAR Models with MolPMoFiT. [ChemRxiv](https://chemrxiv.org/articles/Inductive_Transfer_Learning_for_Molecular_Activity_Prediction_Next-Gen_QSAR_Models_with_MolPMoFiT/9978743/1)
2. Predicting Materials Properties with Little Data Using Shotgun Transfer Learning. [ACS Cent. Sci.](https://pubs.acs.org/doi/full/10.1021/acscentsci.9b00804)
3. Using Rule-Based Labels for Weak Supervised Learning: A ChemNet for Transferable Chemical Property Prediction. [arXiv](https://arxiv.org/abs/1712.02734)
4. Accurate and transferable multitask prediction of chemical properties with an atoms-in-molecules neural network. [Science Advances](https://advances.sciencemag.org/content/5/8/eaav6490). [Github](https://github.com/aiqm/aimnet)
5. Deep Transferable Compound Representation across Domains and Tasks for Low Data Drug Discovery. [J. Chem. Inf. Model.](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.9b00626)
6. An Integrated Transfer Learning and Multitask Learning Approach for Pharmacokinetic Parameter Prediction. [arXiv](https://arxiv.org/abs/1812.09073)
7. Approaching coupled cluster accuracy with a general-purpose neural network potential through transfer learning. [ChemRxiv](https://chemrxiv.org/articles/Outsmarting_Quantum_Chemistry_Through_Transfer_Learning/6744440)
8. SMILES Transformer: Pre-trained Molecular Fingerprint for Low Data Drug Discovery. [https://arxiv.org/abs/1911.04738](https://arxiv.org/abs/1911.04738). [Github](https://github.com/DSPsleeporg/smiles-transformer)
9. Mol2vec: Unsupervised Machine Learning Approach with Chemical Intuition. [J. Chem. Inf. Model.](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.7b00616)
10. Multi-channel PINN: investigating scalable and transferable neural networks for drug discovery. [J. Cheminform.](https://link.springer.com/article/10.1186/s13321-019-0368-1)
11. Distributed Representation of Chemical Fragments. [ACS Omega](https://pubs.acs.org/doi/abs/10.1021/acsomega.7b02045)
12. Pre-training Graph Neural Networks. [arXiv](https://arxiv.org/abs/1905.12265)
13. FP2VEC: a new molecular featurizer for learning molecular properties. [Bioinformatics](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz307/5487389)
14. Self-Attention Based Molecule Representation for Predicting Drug-Target Interaction. [arXiv](https://arxiv.org/abs/1908.06760)
15. DeepAffinity: Interpretable Deep Learning of Compound-Protein Affinity through Unified Recurrent and Convolutional Neural Networks. [arXiv](https://arxiv.org/abs/1806.07537)
16. Pushing the Boundaries of Molecular Representation for Drug Discovery with the Graph Attention Mechanism. [J. Med. Chem.](https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00959)

---

# Materials science 

*More to come*

---

# Credits

Original list taken from:
1. https://github.com/XinhaoLi74/DL-in-Chemistry
2. https://github.com/kangway/mlchempapers
3. https://github.com/getblackbox/Machine-Learning-Potentials-Papers
